disnake/ui/select.py

```diff
@@ -340,7 +362,7 @@ def decorator(func: ItemCallbackType[Select]) -> DecoratedItem[Select]:
             "row": row,
             "min_values": min_values,
             "max_values": max_values,
-            "options": options,
+            "options": _parse_select_options(options),
```

Comment:
```suggestion
            "options": options,
```
This shouldn't be required, since `Select.__init__` already parses the options into the correct format

Reply:
Yup, fixed. Somehow I thought they wouldn't get serialized to a class and then deserialized again, but in hindsight, that doesn't even make sense because the data formats don't quite match up either. Perhaps I should actually check next time :^)